packages/perseus/src/__tests__/item-renderer.test.tsx

```diff
@@ -132,6 +132,13 @@ describe("item renderer", () => {
     });
 
     beforeEach(() => {
+        // Mock ResizeObserver used by the mobile keypad
+        window.ResizeObserver = jest.fn().mockImplementation(() => ({
```

Comment:
Jest throws `ResizeObserver not defined` without this

Reply:
Yup.  Alternatively, you _could_ throw this into the jest setup (`config/test/test-setup.ts`).  Might be a premature optimization though. 🤷‍♂️

Reply:
I did try that initially - took a couple of examples off of StackOverflow but they all failed tests. I'm sure there's a way, but it seemed like a premature optimization 😄